src/card/card.jsx

```diff
@@ -46,6 +53,7 @@ const Card = React.createClass({
 
   getDefaultProps() {
     return {
+      expanded: undefined,
```

Comment:
Can we use `null` as the uncontrolled state. That's for consistency with the other components.